build.gradle

```diff
@@ -716,10 +716,10 @@ subprojects {
             ]
         }
 
-        // Create an allTests task per project
-        tasks.create(name: 'allTests', type: GradleBuild, group: 'Verification') {
-            description 'Run all Checker Framework tests'
-            tasks = ['test', 'checkPurity']
+        // Create an nonCheckerJunitTests task per project
```

Comment:
Because the `junit-tests` job only runs the checker project junit tests. I could change the names to `checker-junit-tests` and `all-tests-but-checker-junit-tests`.

Reply:
Or you could change the junit-tests job to run the junit tests in every project.  I was wondering why you didn't choose to do that.

Reply:
Running all the junit tests takes 36 min and running all non-junit tests only takes 19 min.
Running the checker-junit tests takes 29 min and running all other tests takes 27 min.

Reply:
I see.  That's reasonable, then.  Please make sure what is actually happening is clear (in names and/or comments).  Thanks!